### PR TITLE
docs: Clean up k8s cluster peering instructions

### PR DESCRIPTION
**Pull request by david-yu**
### Description
Small change in consistent style and adding explicit commands. Bump up versions, use minimal values config file. 

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] not a security concern

### CODE REVIEW

File: website/content/docs/connect/cluster-peering/k8s.mdx [hunk 1]
```diff
@@ -82,7 +76,7 @@ Install Consul on Kubernetes by using the CLI to apply `values.yaml` to each clu
       ```
 
       ```shell-session
-    $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.0" --values values.yaml --kube-context $CLUSTER1_CONTEXT
+    $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.1" --values values.yaml --kube-context $CLUSTER1_CONTEXT
```
Comment (kschoche) [inline website/content/docs/connect/cluster-peering/k8s.mdx:79]: nit: It is a little confusing (possibly wrong somewhere?) that we do not set `global.datacenter` for dc1 in the `values.yaml` OR on command line, and then use `--set global.datacenter=dc2` on command line for the dc2 example. I think people might miss this detail, does it make sense to call it out or be explicit in both values.yaml or helm commands?

**Review by kschoche — APPROVED** [review]
Looks great, I left one question but feel free to merge if you disagree!

File: website/content/docs/connect/cluster-peering/k8s.mdx (hunk 1 above)
Comment (david-yu) [inline website/content/docs/connect/cluster-peering/k8s.mdx:79]:
```suggestion
    $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.1" --values values.yaml --set global.datacenter=dc1 --kube-context $CLUSTER1_CONTEXT
```

Comment (david-yu) [reply website/content/docs/connect/cluster-peering/k8s.mdx:79]: I'm ok with changing it to the above, we can change later if needed.

**Pull request merged**